lua/presence/init.lua

```diff
@@ -465,13 +477,8 @@ function Presence:update_for_buffer(buffer, should_debounce)
         small_text = use_file_as_main_image and neovim_image_text or file_text,
     }
 
-    local editing_text = self.options.editing_text
-    editing_text = type(editing_text) == "function"
-        and editing_text(filename, buffer)
-        or string.format(editing_text, filename)
-
     local activity = {
-        state = editing_text,
+        state = self.get_status_text(filename),
```

Comment:
We actually don't need any changes in this function now that we're setting the `get_status_text` method on the `editing_text` option in the `setup` function.

Adding the method call here would actually remove the ability for users to configure their own text. :) You can revert it like this to make the `editing_text` now work as expected:

```suggestion
    local editing_text = self.options.editing_text
    editing_text = type(editing_text) == "function"
         and editing_text(filename, buffer)
         or string.format(editing_text, filename)

    local activity = {
        state = editing_text,
```

Reply:
Oh I see how that works now, just reverted it

Reply:
Oof I think I botched the suggestion 🤦 Can you remove that extranous `local activity = {` text at line 486?

Reply:
Yeah I already got it in the latest commit :)

Reply:
O that was quick, thanks! 🚀